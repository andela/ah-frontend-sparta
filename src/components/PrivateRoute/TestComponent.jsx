import React from 'react';

const TestComponent = () => (
  <>
    <h2>Hello this is a test component</h2>
  </>
);
export default TestComponent;
