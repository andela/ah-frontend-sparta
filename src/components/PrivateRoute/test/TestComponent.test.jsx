import React from 'react';
import renderer from 'react-test-renderer';
import TestComponent from '../TestComponent';

describe('private componet', () => {
  it('should render successfully', () => {
    const props = {
      isLoggedIn: true,
    };
    const wrapper = renderer.create(
      <TestComponent isLoggedIn={props.isLoggedIn} />,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
