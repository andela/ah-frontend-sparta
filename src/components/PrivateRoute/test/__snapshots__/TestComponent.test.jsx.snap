// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`private componet should render successfully 1`] = `
<h2>
  Hello this is a test component
</h2>
`;
